docs/example-scenario/data/realtime-analytics-vehicle-iot.md

```diff
@@ -0,0 +1,95 @@
+author: meeral
+ms.date: 09/12/2018
+---
+
+# Big Data processing pipeline to ingest and process real-time IoT Data from vehicles
+
+This example scenario builds a real time data ingestion and processing pipeline to ingest and process messages from IoT devices (in general sensors) into a big data analytic platform in Azure. Vehicle telematics ingestion and processing platform is the key to create connected car solutions and widely used big data processing pattern in car industries. This specific scenario is motivated by the car telemetry ingestion and processing systems; however, the design patterns are relevant for many industries using sensor-based systems to manage and monitor complex systems in industries such as to requiring advanced analytics of large datasets such as smart buildings, communications, manufacturing, retail, and healthcare etc.
+ 
+This example demonstrates a real time data ingestion and processing pipeline to ingest and process messages from IoT devices (i.e. installed in vehicles). Thousands and millions of messages (or events) are generated by the IoT devices (or sensors), if we can capture these messages and analyze those, we can decipher valuable insights and take appropriate actions. For example, with cars equipped telematics devices, if we can capture the device (IoT) messages in real time and analyze the messages, we would be able to monitor the live location of vehicles, plan optimized routes, providing online or offline assistant to drivers, and supporting telematics-related industries (such as auto insurance).
+
+For this example demonstration, imagine a car manufacturing company, that wants to create a real time telematics ingestion and processing system to ingest and process messages from telematics devices. The company's goals include:
+* Ingest and store data temporarily (i.e. in queue) in real time from vehicles sensors and devices (IoT devices).
+* Analyze the messages in real time to understand vehicle location, and other vehicle information emitted through different types of sensors (i.e. engine related sensors, environment related sensors etc.).
+* Store the data after analysis for other downstream processing to provide actionable insights (i.e. in accident scenarios, insurance agencies may be interested to know what happened during an accident etc.)
+
+
+## Potential use cases
+
+Consider this scenario for the following use cases along with the above goals, when creating telematics ingestion and processing system:
+
+* Vehicle Maintenance Reminders and Alerting.
+* Location based services for the vehicle passengers (i.e. SOS)
+* Autonomous (Self-driving) vehicles.
+
+
+## Architecture
+
+![Screenshot](media/architecture-diagram-realtime-analytics-vehicle-data1.png)
+
+Big Data pipelines in Azure implements Lambda architecture. The solution falls into the speed layer (the layer that processes data in motion) of Lambda architecture. The data flows through the solution as follows:
```

Comment:
If this only implements the speed layer (hot path) then by definition it's not a lambda architecture. You could call it kappa or just real-time stream processing.

Reply:
Strictly speaking I agree with you. I was trying to point to the fact that we can compare the implementation similar to speed layer in lambda architecture. I can elaborate on the fact that, once stored in the serving layer, the batch layer can follow from there, that would be a Lamda architecture context, however the doc then needs to be extended, in my opinion. Not sure if we want to extend further. May be will just keep as "real time big data analytic platform"? Thoughts?

Reply:
Got it, I see what you're saying...  but I see a lot of confusion about lambda architecture, might be better just to call it real-time stream processing or equivalent.

Reply:
Made the changes and committed the update. Can you please review and let me know if this looks good now?
Thanks for your help reviewing the document